newapi/ooniapi/measurements.py

```diff
@@ -1198,6 +1708,175 @@ def coalsum(name):
         return jsonify({"v": 0, "error": str(e)})
 
 
+def _clickhouse_aggregation(
+    resp_format,
+    since,
+    until,
+    inp,
+    domain,
+    category_code,
+    probe_cc,
+    probe_asn,
+    test_name,
+    axis_x,
+    axis_y,
+):
+    log = current_app.logger
+    dimension_cnt = int(bool(axis_x)) + int(bool(axis_y))
+    cacheable = until and until < datetime.now() - timedelta(hours=72)
+    cacheable = False  # FIXME
```

Comment:
Since measurements can also be uploaded at a later stage I recon we should disable caching altogether on these endpoints.

Reply:
The caching is temporarily disabled but could be added back and used only when the msmts are old enough to safely support caching.